help/en/html/tools/logixng/reference/chapter8.shtml

```diff
@@ -340,19 +344,25 @@ WARN  - Log local variables done [JMRI LogixNGThread]
       but another tool is the action <strong>Log local variables</strong>. Each time it's executed,
       it prints all the local variables and their values to the JMRI Sytem Console.</p>
 
+      <div style="margin-left: 2em;">
+        <a href="images/chapter8/log_variables.png"><img src=
+        "images/chapter8/log_variables.png" alt="Chapter 8 log variables"></a>
+      </div>
+
       <p>Options:</p>
 
       <ul>
         <li><strong>Include global variables</strong> &mdash; The global variables will be
         listed after the local variables.</li>
-        <li><strong>Expand arrays and maps</strong> &mdash; the contents of each array and map
+
+        <li><strong>Expand arrays and maps</strong> &mdash; The contents of each array and map
         will be listed.</li>
+
+        <li><strong>Show class name</strong> &mdash; The name of the Java class will be included
+        for each item.</li>
```

Comment:
A suggestion is to add:
`This is useful for debugging when you want to know the actual type of the value.`

Reply:
Done.